app/components/header_component/view.rb

```diff
@@ -18,15 +20,15 @@ def call
       end
     end
 
-  private
+    private
```

Comment:
This conflicts with rubocop-govuk, and seems like purely a style thing

https://github.com/alphagov/rubocop-govuk/blob/7dc2cb6bafa0f605cf86fcf98f1e08505d11f17b/config/layout.yml#L23-L28